dbfit-java/informix/src/main/java/dbfit/environment/InformixBigDecimalTransformer.java

```diff
@@ -0,0 +1,20 @@
+package dbfit.environment;
+
+import java.sql.SQLException;
+
+import dbfit.util.TypeTransformer;
+
+public class InformixBigDecimalTransformer implements TypeTransformer {
+
+    @Override
+    public Object transform(Object o) throws SQLException {
+        if (o == null) {
+            return null;
+        }
+        if (!(o instanceof dbfit.util.NormalisedBigDecimal)) {
+            throw new UnsupportedOperationException("InformixBigDecimalSpecifier cannot work with " + o.getClass());
```

Comment:
Still specifier in the message? (Maybe we can use the class name in the exception message to keep it in sync? /we're renaming too many times/).